minutes/2016-06-24.md

```diff
@@ -0,0 +1,209 @@
+    *   Also, distinguish summary (limit of 77 or 80 chars) from description (unlimited)
+
+*   Google  hangouts has a max capacity of 10. Is it worth considering other  methods of communication so everyone who wants to participate can?
+
+*   Maybe this ( [](http://www.freeconferencecalling.com/)http://www.freeconferencecalling.com/ ) is an option.
+*   Bluejeans 
+*   Continuum has webex.  Past experience is that some Linux platforms had trouble connecting
+
+*   Drop numpy 1.10 and reduce our build matrix. (Numba now works with numpy 1.11.)
+
+*   This comment from the PR for graphviz is the best summary I've seen: [conda forge/staged recipes#568](https://github.com/conda-forge/staged-recipes/pull/568)#issuecomment-225315370
+*   Thanks for pointing this out. The described solution looks reasonable and is preferable to prefixing package names. Great!
+
+*
+
+*   Signing packages
+
+        *   Should be easy to do. ( [](http://conda.pydata.org/docs/signed-packages.html)http://conda.pydata.org/docs/signed-packages.html )
+    *   There has been some interest previously.
+
+*   HTTPError: 503 Server Error: Service Unavailable: Back-end server is at capacity for url...
+
+        *   Seems we are regularly running into this issue under normal usage conditions.
+    *   Had discussed previously caching packages on AppVeyor and trying to reuse those to start.
+    *   Maybe we need to consider caching on all CIs.
+    *   Building our own Miniconda-like self-extracting scripts with packages via [`constructor`](https://github.com/conda/constructor).
+
+**Notes**
+
+Most pressing issues: naming conventions
+
+Naming conventions
+
+*   Continuum's opinion : will take some time for name spaces to take effect, does not want to break anyone's setup, so keep current names, can we follow defaults where defaults have precedent?  Where Continuum does not have packages can they follows conda-forge?
+*   simplegeneric issue, clobbering
+*   how to know what package gets installed when you do a "conda install gplot"? Leads to reproducible environments.
+*   Start with no namespaces, get name-spaces after you install a "core" package (python, r, etc), then you will get packages which match the languages in your environment
+*   Want conda to act like pip, cran, etc, "just works"
+*   What to do about dependencies?
+*   Proposed that when you install a package you will get the packages in all namespaces?
+*   Another option is to specify language in package name (python-simplegeneric), and have lookup table for "common" packages
+*   Should raise issues on conda GitHub repo
+*   No easy solutions, but we need to choose some solution
+*   meta-package which use "common" name
+*   Correct solution is to prefix everything with "python-" but people do not want to do this when installing and people are already used to the old method.
+*   Filipe's issue with namespaces is that it makes choices for users, would rather have that choice... raise on GitHub issue XXX
```

Comment:
IIRC this may be referring to an issue that doesn't exist yet about namespaces. Once one opens (likely on the `conda` issue tracker), I believe @msarahan asked @ocefpaf to mention his thoughts there.